core/src/main/scala/vapors/algebra/Expr.scala

```diff
@@ -28,7 +28,7 @@ import scala.collection.MapView
   * R = Return type
   * P = Captured param
   */
-sealed abstract class Expr[V, R, P] {
+sealed abstract class Expr[V, R, P] {Coursera
```

Comment:
This looks like a bug.

Reply:
Yeah--was trying to clean spam out of my inbox.